tools/mini-yetus.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/sh
```

Comment:
SPDX header, please :)

Reply:
almost none of the files in tools have it, I though that's the tradition :D

Reply:
I don't think it was an intentional tradition =)

Reply:
😃 done.

Reply:
oh no "Linux Foundation".